packages/forma-36-react-components/src/components/Notification/index.tsx

```diff
@@ -57,6 +57,8 @@ const show = (intent: NotificationIntent) => (
     duration?: number;
     canClose?: boolean;
     id?: string;
+    title?: string;
```

Comment:
Hey @suevalov not sure if these changes are needed (here and on line 78), so I wanted to confirm with you. Things still seem to work without them, but I feel weird about not including them without first checking with you.

Reply:
Yep, these changes are needed.